src/components/common/Tabs/TabItem.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import styled from 'styled-components'
+
+interface TabProps {
+  title: string
+  readonly id: number
+  readonly isActive: boolean
+  readonly activeColor?: string
+  onTabClick: (arg: number) => void
+}
+
+interface WrapperProps {
+  readonly activeColor?: string
+}
+
+const Wrapper = styled.li<WrapperProps>`
+  background: var(--color-primary);
+  color: var(--color-text-secondary2);
+  height: var(--height-button-default);
+  display: flex;
+  align-items: center;
+  flex: 1 1 0;
+  font-weight: var(--font-weight-bold);
+  font-size: var(--font-size-large);
+  letter-spacing: 0.1rem;
+  text-align: center;
+  justify-content: center;
+  cursor: pointer;
+
+  &:first-of-type {
+    border-top-left-radius: var(--border-radius-default);
+    border-bottom-left-radius: var(--border-radius-default);
+  }
+
+  &:last-of-type {
+    border-top-right-radius: var(--border-radius-default);
+    border-bottom-right-radius: var(--border-radius-default);
+  }
+
+  &.isActive {
+    background: ${(props): string => (props.activeColor ? `var(${props.activeColor})` : 'var(--color-primary)')};
+    color: var(--color-text-primary);
+  }
+`
+
+const TabItem: React.FC<TabProps> = (props) => {
+  const { onTabClick, id, title, isActive, activeColor } = props
+
+  return (
+    <Wrapper activeColor={activeColor} className={isActive ? 'isActive' : ''} onClick={(): void => onTabClick(id)}>
```

Comment:
@Velenir @anxolin i think we should finalise on this concept:

1. use classes 
2. use styled props

to affect change in styled comps. If `2` is preferred, this could be rewritten as:
```ts
const Wrapper = styled.li.attrs(({ $activeColor, $isActive }) => ({
  $backgroundColor: `var(${$activeColor ? $activeColor : '--color-primary'})`,
  $color: `var(--color-text${$isActive ? '--color-primary' : '--text-secondary2'})`
}))`
  background: ${({ $backgroundColor }): string => $backgroundColor};
  color: ${({ $color }): string => $color};
  height: var(--height-button-default);
  display: flex;
  align-items: center;
  flex: 1 1 0;
  font-weight: var(--font-weight-bold);
  font-size: var(--font-size-large);
  letter-spacing: 0.1rem;
  text-align: center;
  justify-content: center;
  cursor: pointer;

  &:first-of-type {
    border-top-left-radius: var(--border-radius-default);
    border-bottom-left-radius: var(--border-radius-default);
  }
  &:last-of-type {
    border-top-right-radius: var(--border-radius-default);
    border-bottom-right-radius: var(--border-radius-default);
  }
`
```

Reply:
I think this makes sense, given the class was only applied to target it with CSS. So keeping it in the styled component sounds like we can keep styling logic contained.

Reply:
Yes, I vote for `2.` too